packages/browser/src/browser/__tests__/integration.test.ts

```diff
@@ -574,6 +584,34 @@ describe('Dispatch', () => {
     expect(segmentSpy).toHaveBeenCalledWith(boo)
   })
 
+  it('dispatching to Segmentio not blocked by other destinations', async () => {
+    const [ajs] = await AnalyticsBrowser.load({
+      writeKey,
+      plugins: [slowPlugin],
+    })
+
+    const segmentio = ajs.queue.plugins.find((p) => p.name === 'Segment.io')
+    const segmentSpy = jest.spyOn(segmentio!, 'track')
+
+    await Promise.race([
+      ajs.track(
+        'Boo!',
+        {
+          total: 25,
+          userId: '👻',
+        },
+        {
+          integrations: {
+            All: true,
+          },
+        }
+      ),
+      sleep(100),
```

Comment:
What does this sleep do? All promises in a race will continue running even after the first one crosses the finish line -- it's won't abort the promise FYI.

am I missing something -- can we remove this?